module/button.py

```diff
@@ -1,18 +1,61 @@
 from telegram import Update
 from telegram.ext import CallbackContext
+from module.add_book import add_book
+from module.add_item import add_item
 from module.create_connection import create_connection
-from module.shared import DB_PATH
+from module.shared import DB_PATH, error_message
+from module.add_book import add_book
+from module.add_item import add_item
+from module.manage_requests import delete_request, update_similar_requests
 
 
 def button(update: Update, context: CallbackContext) -> None:
     chat_id = update.effective_chat.id
     query = update.callback_query
-    conn = create_connection(DB_PATH)
-    if conn:
+    
+    if query.data.split(';')[0] == 'delete':
+        conn = create_connection(DB_PATH)
+        if conn:
+            cur = conn.cursor()
+            q = query.data.split(';')[1]
+            cur.execute("DELETE FROM Market WHERE rowid=?", (q,))
+            query.edit_message_text(text=f"Eliminazione del libro selezionato...")
+            conn.commit()
+            context.bot.send_message(chat_id, "Libro eliminato.")
+        else:
+            query.edit_message_text(text=f"Si è verificato un problema nella lettura del database.")
+        conn.close()
+    
+    if query.data.split(';')[0] == 'new_request':
+        _, vote, row_id = query.data.split(';')
+        
+        conn = create_connection(DB_PATH)
+        if not conn:
+            context.bot.send_message(chat_id, error_message)
+            return
         cur = conn.cursor()
-        cur.execute("DELETE FROM Market WHERE rowid=?", (query.data,))
-        query.edit_message_text(text=f"Eliminazione del libro selezionato...")
-        conn.commit()
-        context.bot.send_message(chat_id, "Libro eliminato.")
-    else:
-        query.edit_message_text(text=f"Si è verificato un problema nella lettura del database.")
+        cur.execute("SELECT rowid, * FROM Requests WHERE rowid=?", (row_id,))
+        rows = cur.fetchall()
+        
+        if rows:
+
+            if vote == 'Y':
```

Comment:
here and below, avoid magic strings like `'Y'` or `'N'`, save them into constant variable like `NO = 'N'` and `YES = 'Y'`